include/kyosu/functions/acospi.hpp

```diff
@@ -44,8 +40,7 @@ namespace kyosu
 //!   @code
 //!   namespace kyosu
 //!   {
-//!      template<eve::floating_ordered_value T>     constexpr auto acospi(T z) noexcept;  //1
-//!      template<kyosu::concepts::cayley_dickson T> constexpr auto acospi(T z) noexcept;  //2
+//!     constexpr complexify_t<Z> acospi(concepts::cayley_dickson_like auto z) noexcept;
```

Comment:
Read after you do batch changes please...

Reply:
done